src/theme.ts

```diff
@@ -159,8 +160,9 @@ export const chatTheme: ChatTheme = {
   input: {
     base: 'flex mt-4 relative',
     upload: 'px-5 py-2 text-white size-10',
-    input:
+    container:
```

Comment:
I tthink this should probably be `inputContainer` or maybe just do `input` on `inputContainer` and do a sub select

Reply:
updated to use one theme key
`[&>textarea]:w-full`